pyproject.toml

```diff
@@ -81,3 +83,23 @@ find = {}  # Scanning implicit namespaces is active by default
 
 [tool.setuptools.dynamic]
 version = {attr = "pyperformance.__version__"}
+
+[tool.mypy]
+python_version = "3.7"
+pretty = true
+enable_error_code = "ignore-without-code"
+disallow_any_generics = true
```

Comment:
This means that mypy will forbid using, for example `list` as an annotation. You should use `list[int]` instead, which gives the type checker much more information to work with.